dart/lib/src/sentry_options.dart

```diff
@@ -241,7 +259,10 @@ class SentryOptions {
   /// to be sent to Sentry.
   TracesSamplerCallback? tracesSampler;
 
-  SentryOptions({this.dsn, PlatformChecker? checker}) {
+  SentryOptions({this.dsn, PlatformChecker? checker, bool empty = false}) {
+    if (empty) {
+      return;
+    }
```

Comment:
l: This looks very strange to me, a parameter that makes the function ignore all other parameters. I would prefer having an alternative constructor like this:

```suggestion
  SentryOptions.empty();
  
  SentryOptions({this.dsn, PlatformChecker? checker}) {
```

Reply:
We have that on Java, the reason is testing only, so we don't bother to create the objects that are not used, we also have the `empty` factory on Java iirc.